src/app/globalsearch/globalsearch.component.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Search</title>
+</head>
+<body>
+<div class="search-container">
+  <div class="input-group margin-bottom-none">
+    <input type="text" name="text" id="local-searchbox" [(ngModel)]="searchText" placeholder="Search" class="input-group-field" value="">
+    <div class="input-group-button" style="position: relative;">
+      <button mat-button style="position: absolute; right: 0; border: none;" (click)="onSearch()">
+        <mat-icon>search</mat-icon></button>
+    </div>
+  </div>
+</div>
+
+<mat-spinner class="spinner-with-margin" mode="indeterminate" *ngIf="showSpinner" color="primary" diameter="50"></mat-spinner>
+
+<div id="search-results" *ngIf="showResults">
+  <table>
+    <tr *ngFor="let item of jsonData | keyvalue">
+      <td>
+        <p>Index: {{ item.key }}</p>
+      </td>
+      <td>
+        <p>Total Hits: {{ item.value.totalHits }}</p>
+      </td>
+      <td>
+        <p>Link:
+          <a [routerLink]="['/' + item.key ]" [queryParams]="{ searchTerm: searchText }">
+            {{ item.key + '?searchTerm=' + searchText }}
+          </a>
+        </p>
+      </td>
+    </tr>
+  </table>
```

Comment:
I don't we need a table here, can you convert this to a Angular material list please